aframe/pipelines/sandbox/configs/bbh.cfg

```diff
@@ -4,139 +4,20 @@ local_scheduler = true
-workflow = htcondor
-start = &::luigi_base::train_start
-end = &::luigi_base::train_stop
-sample_rate = &::luigi_base::sample_rate
-min_duration = 1024
-max_duration = &::luigi_base::max_duration
-flag = &::luigi_base::flag
-ifos = &::luigi_base::ifos
-channels = &::luigi_base::channels
-request_memory = 6GB
-request_disk = 1024KB
-request_cpus = 1
-
-[luigi_TrainingWaveforms]
-num_signals = 100000
-sample_rate = &::luigi_base::sample_rate
-waveform_duration = &::luigi_base::waveform_duration
-minimum_frequency = &::luigi_base::minimum_frequency
-reference_frequency = &::luigi_base::reference_frequency
-waveform_approximant = &::luigi_base::waveform_approximant
-coalescence_time = &::luigi_base::coalescence_time
-prior = &::luigi_base::prior
-
-[luigi_ValidationWaveforms]
-workflow = htcondor
-num_jobs = 20
-num_signals = 20000
-ifos = &::luigi_base::ifos
-highpass = &::luigi_base::highpass
-sample_rate = &::luigi_base::sample_rate
-waveform_duration = &::luigi_base::waveform_duration
-minimum_frequency = &::luigi_base::minimum_frequency
-reference_frequency = &::luigi_base::reference_frequency
-waveform_approximant = &::luigi_base::waveform_approximant
-coalescence_time = &::luigi_base::coalescence_time
-prior = &::luigi_base::prior
-snr_threshold = 4
-
-[luigi_FetchTest]
-workflow = htcondor
-start = &::luigi_base::train_stop
-end = &::luigi_base::test_stop
-sample_rate = &::luigi_base::sample_rate
-min_duration = 128
-max_duration = &::luigi_base::max_duration
-flag = &::luigi_base::flag
-channels = &::luigi_base::channels
-request_memory = 6GB
-request_disk = 1024KB
-request_cpus = 1
-
 [luigi_TimeslideWaveforms]
-workflow = htcondor
-num_injections = 100000
-shifts = &::luigi_base::shifts
 spacing = 16
 buffer = 16
-snr_threshold = 4
-seed = &::luigi_base::seed
-prior = &::luigi_base::prior
-start = &::luigi_base::train_stop
-end = &::luigi_base::test_stop
-ifos = &::luigi_base::ifos
-psd_length = &::luigi_base::inference_psd_length
-sample_rate = &::luigi_base::sample_rate
-minimum_frequency = &::luigi_base::minimum_frequency
-reference_frequency = &::luigi_base::reference_frequency
-waveform_duration = &::luigi_base::waveform_duration
-waveform_approximant = &::luigi_base::waveform_approximant
-coalescence_time = &::luigi_base::coalescence_time
-highpass = &::luigi_base::highpass
-request_memory = 6GB
-request_disk = 1024KB
-request_cpus = 1
-
-[luigi_SandboxExport]
-fduration = &::luigi_base::fduration
-kernel_length = &::luigi_base::kernel_length
-inference_sampling_rate = &::luigi_base::inference_sampling_rate
-sample_rate = &::luigi_base::sample_rate
-
-# TODO: resolve enum platform parsing error
-# platform = luigi.Parameter(default="TENSORRT")
-ifos = &::luigi_base::ifos
-batch_size = &::luigi_base::inference_batch_size
-psd_length = &::luigi_base::inference_psd_length
-highpass = &::luigi_base::highpass
-
-streams_per_gpu = &::luigi_base::streams_per_gpu
-aframe_instances = 1
-preproc_instances = 1
-clean = true
 
 [luigi_Train]
```

Comment:
And I guess a similar question to the other one: why can't all of these be inherited?